mcp-sdk-functions-hosting-python/.vscode/mcp.json

```diff
@@ -0,0 +1,39 @@
+{
+	"servers": {
```

Comment:
why are there two mcp servers? Do we want to remove one?
<img width="403" height="120" alt="image" src="https://github.com/user-attachments/assets/a802cc10-65e9-4af1-9f5e-b16e34f27847" />